client/src/app-header/AppHeaderUser.js

```diff
@@ -11,9 +11,15 @@ function mapStateToProps(state) {
 const Connected = connect(mapStateToProps)(React.memo(UserButton));
 
 function UserButton({ currentUser }) {
-  return (
-    <div className={styles.style}>{currentUser.name || currentUser.email}</div>
-  );
+  if(currentUser._id === "noauth") {
```

Comment:
Style nit, add a space after `if`.

Reply:
Regarding all superficial style issues like whitespace, positioning of braces, etc, SQLPad uses `prettier`. In the top level, there are `fixlint` and `lint` scripts, so you can do `npm run lint` and `npm run fixlint` -- the latter will rewrite the files.

Reply:
Good to know! Apply the linter for all files.